docs/features/dns-cache.rst

```diff
@@ -31,7 +34,7 @@ of next_node.name may be resolved even without upstream connectivity.
   },
 
   next_node = {
-    name = 'nextnode',
+    name = 'nextnode.yourdomain.net',
```

Comment:
I quite like it now. one minor style-thing: short forms "it's", "doesn't" usually is not used in written language